test/unit/request-auth.test.js

```diff
@@ -123,4 +123,29 @@ describe('RequestAuth', function () {
             fakeParent: true
         });
     });
+
+    describe('parameters', function () {
+        it('should return with already set auth params if type set to invalid auth type', function () {
```

Comment:
```suggestion
	describe('.update', function () {
        it('should not update auth params if type is invalid', function () {
```

Reply:
Moving request-auth changes to separate PR as suggested by @codenirvana https://github.com/postmanlabs/postman-collection/pull/1101